test/SignalRServiceExtension.Tests/SignalRAsyncCollectorTests.cs

```diff
@@ -35,7 +41,7 @@ public async Task AddAsync_WithBroadcastMessage_CallsSendToAll()
             Assert.Equal("newMessage", actualData.Target);
             Assert.Equal("arg1", actualData.Arguments[0]);
             Assert.Equal("arg2", actualData.Arguments[1]);
-            Assert.Equal(Endpoints, actualData.Endpoints);
+            Assert.Equal(endpoints, actualData.Endpoints);
```

Comment:
Suppose not setting Endpoints equals selecting all endpoints, why the validation is different?

Reply:
Also covers endpoints routing changes default values?